### ISSUE TITLE
AccessViolationException in RuntimeHelpers.PrepareMethod(System.Runtime.Intrinsics.X86.Sse2.MoveScalar)

### ISSUE THREAD

**Issue by LadislavLang** [dotnet/coreclr#22874]
Tested with targetframework netcoreapp2.2

```c#
using System;
using System.Linq;
using System.Runtime.CompilerServices;

namespace ConsoleApp21
{
    class Program
    {
        static void Main(string[] args)
        {
            var methodInfoList = Type.GetType("System.Runtime.Intrinsics.X86.Sse2").GetMethods().Where(m => m.Name == "MoveScalar");
            foreach (var methodInfo in methodInfoList)
            {
                // 2nd and 3rd overload throws AccessViolationException
                RuntimeHelpers.PrepareMethod(methodInfo.MethodHandle);
            }
        }
    }
}
```

**Comment by AndyAyersMS** [comment]
x86 intrinsics were in preview form in 2.2, so not officially supported. They will be supported in 3.0, and your example works fine there.

**Comment by AndyAyersMS** [comment]
@LadislavLang this is fixed in 3.0, and we don't plan on fixing the preview bits ... can we close this issue?
